src/Compilers/Server/ServerShared/ServerDispatcher.cs

```diff
@@ -236,6 +236,8 @@ private void HandleCompletedGCTask()
         {
             _gcTask = null;
```

Comment:
What API are you looking to call for this?  Not familiar with how to invoke this.